app/MainHie.hs

```diff
@@ -50,17 +47,14 @@ plugins includeExamples = pluginDescToIdePlugins allPlugins
                    else basePlugins
     basePlugins =
       [ applyRefactDescriptor "applyrefact"
-      , baseDescriptor        "base"
       , brittanyDescriptor    "brittany"
       , haddockDescriptor     "haddock"
       -- , hareDescriptor        "hare"
-      , hoogleDescriptor      "hoogle"
       , hsimportDescriptor    "hsimport"
       , liquidDescriptor      "liquid"
       , packageDescriptor     "package"
       , pragmasDescriptor     "pragmas"
       , floskellDescriptor    "floskell"
-      , biosDescriptor        "bios"
       , genericDescriptor     "generic"
```

Comment:
Why are we removing the base, hoogle and bios plugins?

Reply:
The base plugin was only used by the JSON transport to query existing plugins, LSP can't access them

```
      [ PluginCommand "version" "return HIE version" versionCmd
       , PluginCommand "plugins" "list available plugins" pluginsCmd
       , PluginCommand "commands" "list available commands for a given plugin" commandsCmd
       , PluginCommand "commandDetail" "list parameters required for a given command" commandDetailCmd
```

Same goes for Hoogle and Bios, they only exposed commands that LSP can't access, so it didn't make much sense to keep them around as plugins